src/Features/Core/Portable/PullMemberUp/Dialog/PullMemberUpWithDialogCodeAction.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp.Dialog;
+using Microsoft.CodeAnalysis.PullMemberUp;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp
+{
+    internal abstract partial class AbstractPullMemberUpRefactoringProvider
+    {
+        internal class PullMemberUpWithDialogCodeAction : CodeActionWithOptions
+        {
+            private readonly ISymbol _selectedMember;
```

Comment:
i don't understand what this means.  If you're pulling members up... how is there a selected member?

Reply:
So this field is used because when the dialog is created, the member user clicked will be checked by default.

Reply:
this is worth havign a comment on :)